templates/zerver/app/index.html

```diff
@@ -51,7 +51,7 @@
     color: hsl(0, 0%, 20%);
     }
     :root.dark-theme #app-loading {
-    background-color: hsl(212, 28%, 18%);
+    background-color: hsl(0, 0%, 11%);
```

Comment:
This needs to be duplicated into the corresponding `.color-scheme-automatic` block for the rebase on #25504.